### PR TITLE
Obsoletes RFC6962.

### PR DESCRIPTION
**Pull request by robstradling**

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -134,13 +135,14 @@ informative:
...
 --- abstract
 
-This document describes a protocol for publicly logging the existence of
-Transport Layer Security (TLS) server certificates as they are issued or
-observed, in a manner that allows anyone to audit certification authority (CA)
-activity and notice the issuance of suspect certificates as well as to audit the
-certificate logs themselves. The intent is that eventually clients would refuse
-to honor certificates that do not appear in a log, effectively forcing CAs to
-add all issued certificates to the logs.
+This document describes version 2.0 of the Certificate Transparency (CT)
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: That is one long sentence.

**Comment by robstradling** [comment]
Merged at ebadc0be42e844acb56ece616f31df77ddb7d71e

File: draft-ietf-trans-rfc6962-bis.md (hunk 1 above)
Comment (letoams) [inline draft-ietf-trans-rfc6962-bis.md]: I would actually do:

title: "Certificate Transparency Version 2.0"

That is, no brackets. That matches for instance RFC 7296 for IKE version 2

**Comment by eranmes** [comment]
On Sun, Dec 4, 2016 at 10:36 PM, Paul Wouters (libreswan) <
notifications@github.com> wrote:

> *@letoams* commented on this pull request.
> ------------------------------
>
> In draft-ietf-trans-rfc6962-bis.md
> <https://github.com/google/certificate-transparency-rfcs/pull/207>:
>
> > @@ -134,13 +135,14 @@ informative:
>
>  --- abstract
>
> -This document describes a protocol for publicly logging the existence of
> -Transport Layer Security (TLS) server certificates as they are issued or
> -observed, in a manner that allows anyone to audit certification authority (CA)
> -activity and notice the issuance of suspect certificates as well as to audit the
> -certificate logs themselves. The intent is that eventually clients would refuse
> -to honor certificates that do not appear in a log, effectively forcing CAs to
> -add all issued certificates to the logs.
> +This document describes version 2.0 of the Certificate Transparency (CT)
>
> I would actually do:
>
> title: "Certificate Transparency Version 2.0"
>
> That is, no brackets. That matches for instance RFC 7296 for IKE version 2
>
https://github.com/google/certificate-transparency-rfcs/pull/211

**Pull request closed without merging**